examples/product/config/sw-config.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
       handler: "fastest",
       urlPattern: "^https:\/\/fonts\.gstatic\.com"
     }],
-    staticFileGlobs: ['dist/**/*']
+    cacheId: "electrodeProgressiveProduct",
+    importScripts: ['./sw-events.js'],
```

Comment:
Even though we don't have any push events set up, we need this here until the fixed app archetype PR lands.